src/module/env/index.ts

```diff
@@ -0,0 +1,3 @@
+export function APIEndpoint(): string {
+  return "http://127.0.0.1:7777";
+}
```

Comment:
I would like to establish a single package responsible for providing configuration dynamically configured via the environment.